src/components/pages/Dashboard/ServiceMap.js

```diff
@@ -41,7 +45,19 @@ export default function ServiceMap(props) {
           longitude={dot.longitude}
           latitude={dot.latitude}
         >
-          <img src={yellowPing} />
+          {/* <img src={yellowPing} /> */}
```

Comment:
Would we still need the <img...> when the map is zoomed in and we can see singular dots and not clusters?

Reply:
Yes, right, I am working on V.2 of this now.